docs/manual/java/guide/services/ServiceClients.md

```diff
@@ -59,11 +59,13 @@ All service calls with Lagom service clients are by default using circuit breake
 
 In the above example the default identifier is used for the `sayHi` method, since no specific identifier is given. The default identifier is the same as the service name, i.e. `"hello"` in this example. The `hiAgain` method will use another circuit breaker instance, since `"hello2"` is specified as circuit breaker identifier.
 
+### Circuit Breaker Configuration
+
 On the client side you can configure the circuit breakers. The default configuration is:
 
 @[circuit-breaker-default](../../../../../service/core/client/src/main/resources/reference.conf)
 
-That configuration will be used if you don't define any configuration yourself.
+That configuration will be used if you don't define any configuration yourself. You may want to configure the circuit breaker for a given endpoint to ignore a certain exceptions. Lagom's client [[maps all 4xx and 5xx responses to Exceptions|ServiceErrorHandling]] and Lagom's Circuit Breaker defaults to considering all Exceptions as failures.  This is particularly useful when connecting to services where 4xx HTTP status codes are used to model business valid cases. For example, it may be a non-failure case to respond a 404 to a particular request. In that case the client will still receive a `NotFound` exception as a result of invoking the service but you can add `"com.lightbend.lagom.scaladsl.api.transport.NotFound"` to the circuit breaker whitelist so that it is not considered by the circuit breaker failure count.
```

Comment:
A couple of confusing points:

1) Since it is the default, does the user have to put the notation @circuit-breaker-default  in the client, or is it just used by Lagom even if not specified? This should be made clear at the beginning of the section.
2) First the paragraph talks about the default configuration, then it talks about "Lagom's client" and "Lagom's Circuit Breaker" Does this discussion also include a client that the user might have created? The difference/relationship between these needs to be made more clear as well.
3) Once that is done, I suggest re-ordering the flow of info like this to make it easier to follow, separating the discussion of the default behavior from why/how to change that behavior:
That configuration will be used if you don't define any configuration yourself. Lagom's client [[maps all 4xx and 5xx responses to Exceptions|ServiceErrorHandling]] and Lagom's Circuit Breaker defaults to considering all Exceptions as failures.  This is particularly useful when connecting to services where 4xx HTTP status codes are used to model business valid cases. 

You may want to configure the circuit breaker for a given endpoint to ignore a certain exceptions. For example, it may be a non-failure case to respond a 404 to a particular request. In that case the client will still receive a `NotFound` exception as a result of invoking the service but you can add `"com.lightbend.lagom.scaladsl.api.transport.NotFound"` to the circuit breaker whitelist so that it is not considered by the circuit breaker failure count.

Also, FYI, I tried to read this whole page to get the context for the section you added, and found the page extremely difficult to understand.

Reply:
Thanks @rstento for reviewing this. Let me comment each item separately:

1. the `@[name](file/path.txt) is code to instruct the site generator that some snippet must be inserted here. That is not something that users would really see in the docs. Users browsing the docs in the website will actually see java cod or scala code or, in this case, a piece of `reference.conf`
2. Hmmm, I'm not sure I follow I tried to add cross-references but since this paragraph is in the Lagom clients page I thought there was enough context already. I think in previous paragraphs the concept of a circuit breaker as a building block of a client is already introduced.
3. Thanks, I see the wording is not great but took a slightly different approach.

The client management in lagom and the circuit breaking pattern are not a easy to explain concept. I think we could open a different issue/PR for a page-wide rewrite.